netgrasp/netgrasp.py

```diff
@@ -10,17 +10,19 @@
 import logging
 import logging.handlers
 import pwd
-import sys
 import os
 import datetime
 import time
-import traceback
 
 netgrasp_instance = None
 
 BROADCAST = 'ff:ff:ff:ff:ff:ff'
 
-ALERT_TYPES = ['requested_ip', 'first_requested_ip', 'first_requested_ip_recently', 'seen_device', 'first_seen_device', 'first_seen_device_recently', 'seen_mac', 'first_seen_mac', 'seen_ip', 'first_seen_ip', 'seen_host', 'first_seen_host', 'seen_vendor', 'first_seen_vendor', 'device_stale', 'request_stale', 'changed_ip', 'duplicate_ip', 'duplicate_mac', 'network_scan', 'ip_not_on_network', 'src_mac_broadcast', 'requested_self']
+ALERT_TYPES = ['requested_ip', 'first_requested_ip', 'first_requested_ip_recently', 'seen_device', 'first_seen_device',
+               'first_seen_device_recently', 'seen_mac', 'first_seen_mac', 'seen_ip', 'first_seen_ip', 'seen_host',
+               'first_seen_host', 'seen_vendor', 'first_seen_vendor', 'device_stale', 'request_stale', 'changed_ip',
+               'duplicate_ip', 'duplicate_mac', 'network_scan', 'ip_not_on_network', 'src_mac_broadcast',
+               'requested_self']
```

Comment:
My general concerns:

1. Editors can gracefully handling wrapping of long lines, so this is not essential and is specific to your local setup (I'm using vim, which has no problems with this, for example);
2. It needs to be consistent, but there's lot of long lines in the code, not all wrapped;
3. In Python you can cause bugs by wrapping/indenting incorrectly, so added risk for little-to-no gain.